exercises/reverse-string/tests/reverse-string.rs

```diff
@@ -62,3 +62,11 @@ fn test_a_palindrome() {
 fn test_wide_characters() {
     process_reverse_case("子猫", "猫子");
 }
+
+#[test]
+#[ignore]
+/// bonus only
```

Comment:
Instead of changing the example, introducing a dependency in `Cargo.toml`, etc, why not use conditional compilation to enforce that this is a bonus feature? I.e.

```rust
#[test]
#[ignore]
#[cfg(feature="graphemes")]
fn test_grapheme_clusters() { ... }
```

Examples of how to use custom features [here](https://stackoverflow.com/a/27634313/504550).

Reply:
You'd need to add a comment explaining to students how to run this test.